packs/bitbucket/rules/post_receive_webhook.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: "post_receive_webhook"
+pack: "bitbucket"
+description: "Trigger ST2 pack deploy for commit on Git branch"
+enabled: true
```

Comment:
Also, this rule is a bit of a special case since it relies on functionality from two packs (there are many like that of course, but they tend to be installation specific).

We don't really have a good established pattern for it yet, but maybe it would be better if it's part of `packs` pack. In any case, I will also wait for others to chime in.

/cc @lakshmi-kannan @manasdk